corehq/messaging/tasks.py

```diff
@@ -40,6 +45,17 @@ def sync_case_for_messaging_rule(self, domain, case_id, rule_id):
         self.retry(exc=e)
 
 
+@no_result_task(serializer='pickle', queue=settings.CELERY_REMINDER_CASE_UPDATE_QUEUE, acks_late=True,
+                default_retry_delay=5 * 60, max_retries=12, bind=True)
```

Comment:
nit: I guess since we are handling all exceptions and adding another task for each case that fails, there won't be a retry for this task as a whole ever? But still keeping this task with retry parameters for other scenarios?

Reply:
I guess we can remove it altogether, since the retries are handled serially with the other task which has retries configured. I will remove this